{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "60498814",
  "multipleFaultNumbers": null,
  "date": "2020-12-16T20:13:43",
  "region": "Ulverston",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-16T20:13:43",
  "information": "",
  "actualTimeOfRestoration": "2020-12-16T20:13:43",
  "outageCentrePoint": {
    "lat": 54.19621,
    "lng": -3.10007
  },
  "outageLocations": null,
  "AffectedPostcodes": " LA12 7BD, LA12 7BE, LA12 7BG, LA12 7XA",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}